### ISSUE TITLE
Integrate native Nightingale media controls with media streamed through the web browser

### ISSUE THREAD

**Issue by zjays** [nightingale-media-player/nightingale-hacking#359]
Add the ability to use the native Nightingale media controls (play/pause, previous, next, etc.) to control media that is streamed through the web browser (Youtube, Pandora, etc.)

There is a Firefox add-on that adds media key support for popular media streaming websites, so perhaps it can be adapted into Nightingale in order to help achieve this feature. 
https://addons.mozilla.org/en-us/firefox/addon/media-keys/
https://github.com/carlin-q-scott/browser-media-keys

(This probably won't be very useful until Nightingale uses a more up-to-date web browser, since a lot of streaming web sites don't seem to work properly in Nightingale right now)
##

**Comment by freaktechnik** [comment]
Yes, Nightingale currently doesn't support HTML5 audio and video elements well.

I thought there already was a note for such functionality somewhere, but apparently only on the wiki :)
